src/core/fr-command-7z.c

```diff
@@ -345,7 +345,10 @@ fr_command_7z_add (FrCommand     *comm,
 	if (spd_support) fr_process_add_arg (comm->process, "-spd");
 	fr_process_add_arg (comm->process, "-bd");
 	fr_process_add_arg (comm->process, "-y");
+#ifndef USE_7Z
+	/* 7z does not support '-l'. Define USE_7Z to disable '-l'. */
```

Comment:
Since this is about the difference between 7zip and p7zip, please change it to:

```c++
	/* Unlike p7zip, 7zip does not support '-l'. Define USE_7Z to disable '-l'. */
```

Reply:
done